ReactSkia/core_modules/RSkInputEventManager.h

```diff
@@ -16,36 +18,40 @@ namespace facebook {
 namespace react {
 
 using namespace SpatialNavigator;
-
-struct RskKeyInput {
-  RskKeyInput() = default;
-  ~RskKeyInput() = default;
-  RskKeyInput(rnsKey key, rnsKeyAction keyAction, bool keyRepeat)
+struct RSkKeyInput {
+  RSkKeyInput() = default;
+  ~RSkKeyInput() = default;
+  RSkKeyInput(rnsKey key, rnsKeyAction keyAction, bool keyRepeat)
       : key_(key),
         action_(keyAction),
         repeat_(keyRepeat) { }
   rnsKey key_;
   rnsKeyAction action_ {RNS_KEY_UnknownAction};
   bool repeat_ {false};
 };
+typedef std::function< void (RSkKeyInput)> inputEventClientCallback;
 
 class RSkInputEventManager {
  private:
+  std::mutex eventCallbackMutex_;
+  int callbackId_ = 0;
+  std::map <int, inputEventClientCallback > eventCallbackMap_;
```

Comment:
Can we use unordered map, as it is been mostly used in reactSkia , considering less overhead on insertion.